CHANGELOG.md

```diff
@@ -3,6 +3,46 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## dev
+
+### Added
+
+### Changed
+
+- [#224](https://github.com/nf-core/rnavar/pull/224) - Update ensemblvep modules to 114.2
+- [#224](https://github.com/nf-core/rnavar/pull/224) - Update ensemblvep cache version to 114
+
+### Fixed
+
+- [#224](https://github.com/nf-core/rnavar/pull/224) - bcfann is now a valid tool to annotate vcf files with (BCFTOOLS_ANNOTATE)
+
+### Dependencies
+
+| Dependency | Old version | New version |
+| ---------- | ----------- | ----------- |
+| ensemblvep | 113.0       | 114.2       |
+
+### Parameter
+
+| Old name                  | New name |
+| ------------------------- | -------- |
+| use_annotation_cache_keys |          |
```

Comment:
This is a user deprecation? I would state this so under 'Deprecated' or something similar

Reply:
actually it was already deprecated